ngrinder-controller/src/main/java/org/ngrinder/infra/config/WebMvcConfig.java

```diff
@@ -37,14 +36,19 @@
 	@Value("${server.default-encoding}")
 	private String defaultEncoding;
 
-	@Autowired
-	private LocaleChangeInterceptor localeChangeInterceptor;
+	private final LocaleChangeInterceptor localeChangeInterceptor;
 
-	@Autowired
-	private DefaultSuccessJsonInterceptor defaultSuccessJsonInterceptor;
+	private final DefaultSuccessJsonInterceptor defaultSuccessJsonInterceptor;
 
-	@Autowired
-	private ResourceProperties resourceProperties = new ResourceProperties();
+	private final ResourceProperties resourceProperties;
+
+	public WebMvcConfig(LocaleChangeInterceptor localeChangeInterceptor,
+						DefaultSuccessJsonInterceptor defaultSuccessJsonInterceptor,
+						ResourceProperties resourceProperties) {
+		this.localeChangeInterceptor = localeChangeInterceptor;
+		this.defaultSuccessJsonInterceptor = defaultSuccessJsonInterceptor;
+		this.resourceProperties = resourceProperties;
```

Comment:
I highly recommend to use @RequiredArgsConstructor

Reply:
@junoyoon 

Fixed